crates/index-scheduler/src/snapshots/lib.rs/swap_indexes/third_empty_swap_processed.snap

```diff
@@ -89,7 +89,7 @@ succeeded [0,1,2,3,4,5,6,]
 ----------------------------------------------------------------------
 ### Batches Kind:
 "indexCreation" [0,1,2,3,]
-"indexSwap" [4,5,6,]
+"indexSwap" [4,5,]
```

Comment:
I'm not sure I get the change here, could you please explain it as a response to this comment?

Reply:
Good catch, that was actually a bug.
We handle the empty index swap tasks in another place from the normal index swap. I did not remember that, and I was just missing the task :sweat: 
It's fixed now, thanks :+1: